app/uk/gov/hmrc/contactadvisors/views/Home.scala.html

```diff
@@ -16,7 +16,7 @@
 
 @this(layout: Layout)
 
-@(string: String)(implicit request: Request[_], messages: Messages)
+@(string: String)(implicit request: Request[?], messages: Messages)
```

Comment:
Does this work?

Reply:
@ardwalker - that was one of the warning due to change in wildcard args - https://docs.scala-lang.org/scala3/reference/changed-features/wildcards.html